Feature/add qubicj docs

georgmittendorfer opened the pull request:
Add page for Qubic Java Library (qubicj/qubicj-computor-api) to developers/integration section.
Add page for Qubic Shell (qubicj-shell) to developers/client section.

J0ET0M commented:
thanks, should we call it better "Java Qubic Shell" or "Qubic Java" or similar.
for all the others (except core utils) we name the coding language.
![image](https://github.com/user-attachments/assets/d3f23f3b-32c6-4d24-9a39-580ac8d1061f)

georgmittendorfer commented:
Fixed. Qubic Shell -> Qubic Java Shell.

And I added the info that this is a community project so that it does not get confused with core cli and wallets.
